packages/amplify-dotnet-function-template-provider/resources/lambda/global.json

```
@@ -1,6 +1,6 @@
 {
   "sdk": {
-    "version": "6.0.300",
+    "version": "6.0.100",
     "rollForward": "latestFeature"
```

these settings together mean that customer can use .net6 that's between [6.0.100 , 7), i.e. any 6.x version after 6.0.100

---

Does current code `version": "6.0.300"`, throw error on existing ubuntu based dot net projects ?

---

Then if you install from MSFT feeds (which is more work and recommended way) you'll get successful function build.
The change is meant to cover both cases.

---

to answer the question.
if existing project works on ubuntu then either user had to install dotnet from MS feed or make change after generating function from template to allow 6.0.1xx.

---

Shall we do a strict checking here to make sure, existing functions build fails fast on existing projects  
https://github.com/aws-amplify/amplify-cli/blob/e914703261c8d4855b09cc0a5bc01a7b5de44591/packages/amplify-dotnet-function-runtime-provider/src/utils/detect.ts#L24. and  we can recommend the way to install ?